packages/client/src/Components/PaperEdits/PaperCutsPlayer/Player.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import VideoContext from 'videocontext';
+
+const getCanvas = (ref, width, height) => {
+  return (
+    <canvas
+      ref={ ref }
+      className='video-canvas'
+      width={ width }
+      height={ height }
+    >
+    </canvas>
+  );
+};
+
+class PaperCutsPlayer extends React.PureComponent {
+  constructor(props) {
+    const {
+      width = 640,
+      height = 360,
+      playlist = [],
+    } = props;
+
+    super(props);
+
+    this.playlist = playlist;
+    this.canvasRef = React.createRef();
+    this.canvas = getCanvas(this.canvasRef, width, height);
+  }
+
+  componentDidMount() {
+    this.videoContext = new VideoContext(this.canvasRef.current, () => console.log('oops'));
```

Comment:
Not sure I get what's going on here, VideoContext takes a callback on initialisation? 
or if it if fails to initialise it then triggers the callback?

In any case perhaps this 
```js
console.log('oops')
```

should be a `console.error`? with a more descriptive message? 
Or/And do we need to think about some other error handling/fallback if VideoContext fails to initialise?

Reply:
Aha good catch, I think I put this in for local debugging